packages/mui-material-next/src/TablePagination/TablePagination.test.js

```diff
@@ -128,14 +128,14 @@ describe('<TablePagination />', () => {
       );
 
       // will be `getByRole('combobox')` in aria 1.2
-      const [combobox] = getAllByRole('button');
-      expect(combobox).toHaveAccessibleName('lines per page: 10');
+      const [combobox] = getAllByRole('combobox');
+      expect(combobox).toHaveAccessibleName('lines per page:');
```

Comment:
@DiegoAndai I want to double check with you to see if the change here is expected.

Reply:
Yes, this seems correct as the name shouldn't include the selected value 👍🏼